docker/kyuubi-pod.yaml

```diff
@@ -17,10 +17,13 @@
 
 # Usage:
 # Start:
-#   kubectl apply -f https://raw.githubusercontent.com/apache/incubator-kyuubi/master/docker/example.yaml
+#   kubectl apply -f https://raw.githubusercontent.com/apache/incubator-kyuubi/master/docker/kyuubi-pod.yaml
+# Expose port:
+#  1. for tmp test
+#    kubectl port-forward kyuubi-example --address localhost 10009:10009
+#  2. use NodePort
+#    see usage of kyuubi-service.yaml
```

Comment:
It seems 24-25 line should move to line 30 ?

Reply:
Thanks for your review, it's seems better to put it there